src/v/cloud_storage/base_manifest.h

```diff
@@ -57,9 +57,6 @@ class base_manifest {
     /// \return asynchronous input_stream with the serialized json
     virtual ss::future<serialized_data_stream> serialize() const = 0;
 
-    /// Manifest object format and name in S3
-    virtual remote_manifest_path get_manifest_path() const = 0;
```

Comment:
curious about this one, why is it removed?

Reply:
Some manifests now fully have the ability to generate their paths, but some will rely on the path provider. An alternative is to lean into this more and make the partition manifest have a path provider as a member, but as discussed, this isn't a great option given all the places `partition_manifest` is constructed with simple args (no args, or just an NTPR)